src/ray/gcs/gcs_server/gcs_actor_scheduler.cc

```diff
@@ -370,9 +370,9 @@ void GcsActorScheduler::CreateActorOnWorker(std::shared_ptr<GcsActor> actor,
 
 void GcsActorScheduler::RetryCreatingActorOnWorker(
     std::shared_ptr<GcsActor> actor, std::shared_ptr<GcsLeasedWorker> worker) {
-  execute_after(io_context_,
-                [this, actor, worker] { DoRetryCreatingActorOnWorker(actor, worker); },
-                RayConfig::instance().gcs_create_actor_retry_interval_ms());
+  execute_after(
+      io_context_, [this, actor, worker] { DoRetryCreatingActorOnWorker(actor, worker); },
```

Comment:
Are you using clang-format 7.0? I remember this style lint was applied when I used clang-format 9.0, and I think we are using 7.0 in the CI (I am asking it because this version conflict caused a trouble once to me)

Reply:
Yeah, probably have that issue here now that i've fixed it for the lint script. :)